src/utils/env.js

```diff
@@ -1,19 +1,58 @@
-// Info about the current environment
+/**
+ * Utilities to get information about the current environment
+ */
 
-// Constants
+// --- Constants ---
 
-export const inBrowser = typeof document !== 'undefined' && typeof window !== 'undefined'
-
-export const isServer = !inBrowser
+export const hasWindowSupport = typeof window !== 'undefined'
```

Comment:
Could use the isDefined() helper method here

Reply:
This is more safe and we can't vice versa import between the two files without issues.